module-2/backend/week_one.md

```diff
@@ -5,30 +5,84 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+  get - retrieve a view
+  post - update a portion of the view with the given parameter
+  put - generate new data and return it in the view.
+  delete - delete information
+  
 2. What is Sinatra?
+  Sinatra is a framework language for web development.  My understanding is that it lays out the routes
+  and structure for all of the components to interact with each other.
+  
 4. What is MVC?
+  Model View Controller.  I'm uncertain if this refers to all of them as a concept or just the controller.  The controller's        purpose is to delineate the actions that are taken whenever a request is made.

```

Comment:
Refers to the concept. Looks like you have a high level understanding of the responsibilities of the controller, but would recommend that you look into what the Views and Models do as well. We'll be working with this pattern for the remainder of M2 and into M3.